src/SFA.DAS.EmployerIncentives.Domain/Interfaces/IIncentivePaymentProfilesService.cs

```diff
@@ -6,6 +6,6 @@ namespace SFA.DAS.EmployerIncentives.Domain.Interfaces
 {
     public interface IIncentivePaymentProfilesService
     {
-        Task<IEnumerable<IncentivePaymentProfile>> Get();
+        Task<IncentivesConfiguration> Get();
```

Comment:
This interface doesn't need to change.  The parent of the data is still an IncentivePaymentProfile object.  Extra data has been added to it and some moved around.